stress/shell/src/main/java/alluxio/stress/cli/BenchPlan.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.cli;
+
+/**
+ * Base class for Bench plan.
+ */
+public abstract class BenchPlan {
```

Comment:
There's a package `alluxio.job.plan` already. Maybe you don't want to reuse the word "Plan"?